plugins/woocommerce/src/Internal/Features/FeaturesController.php

```diff
@@ -90,32 +90,37 @@ class FeaturesController {
 	 */
 	public function __construct() {
 		$features = array(
-			'analytics'              => array(
+			'analytics'                     => array(
 				'name'               => __( 'Analytics', 'woocommerce' ),
 				'description'        => __( 'Enables WooCommerce Analytics', 'woocommerce' ),
 				'is_experimental'    => false,
 				'enabled_by_default' => true,
 				'disable_ui'         => false,
 			),
-			'new_navigation'         => array(
+			'new_navigation'                => array(
 				'name'            => __( 'Navigation', 'woocommerce' ),
 				'description'     => __( 'Adds the new WooCommerce navigation experience to the dashboard', 'woocommerce' ),
 				'is_experimental' => false,
 				'disable_ui'      => false,
 			),
-			'new_product_management' => array(
+			'new_product_management'        => array(
 				'name'            => __( 'New product editor', 'woocommerce' ),
 				'description'     => __( 'Try the new product editor (Beta)', 'woocommerce' ),
 				'is_experimental' => true,
 				'disable_ui'      => false,
 			),
-			'custom_order_tables'    => array(
+			'new_product_management_blocks' => array(
+				'name'            => __( 'New product management experience', 'woocommerce' ),
+				'description'     => __( 'Try the new product management experience with Gutenberg Blocks (Alpha)', 'woocommerce' ),
+				'is_experimental' => true,
+			),
```

Comment:
I think on line 417: https://github.com/woocommerce/woocommerce/blob/87f51d9b5dfde76bd3d36dad4b4a9088880d98a6/plugins/woocommerce/src/Internal/Features/FeaturesController.php#L417-L419
You either want to update that condition to use `new_product_management_blocks` or add an or case. At-least that way the current feature flag that we toggle using the beta tester still works.

Reply:
We probably also want to update this condition:
https://github.com/woocommerce/woocommerce/blob/87f51d9b5dfde76bd3d36dad4b4a9088880d98a6/plugins/woocommerce/src/Internal/Features/FeaturesController.php#L665-L668

So the blocks feature is also disabled by default (for now).